.github/workflows/workflow.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and deploy
+
+on:
+  push:
+    branches:
+      - "*"
```

Comment:
Doesn't this mean that it will be triggered on any pushed branch?

Reply:
Yes

Reply:
But shouldn't it be triggered only by pushing a tag?

Reply:
I see, build all branches, release only tags.